2.7/s2i/bin/run

```diff
@@ -13,7 +15,21 @@ function should_migrate() {
   is_django_installed && [[ -z "$DISABLE_MIGRATE" ]]
 }
 
-set -e
+# Guess the number of workers according to the number of cores
+function get_default_web_concurrency() {
+  limit_vars=$(cgroup-limits)
+  local $limit_vars
+  if [ -z "${NUMBER_OF_CORES:-}" ]; then
+    echo 1
+    return
+  fi
+
+  local max=$((NUMBER_OF_CORES*4))
+  local default=$(((${MEMORY_LIMIT_IN_BYTES:-MAX_MEMORY_LIMIT_IN_BYTES}/1024/1024 - 40) / 3))
```

Comment:
my comment seems to have gotten lost, sorry if you get this twice, but:  please update this w/ a comment indicating what this calculation is doing (looks like it assumes 40megs of overhead, then 3 megs per worker?)